lib/elixir_boilerplate_web/endpoint.ex

```diff
@@ -9,6 +9,7 @@ defmodule ElixirBoilerplateWeb.Endpoint do
   )
 
   plug(ElixirBoilerplateWeb.Health.Plug)
+  plug(:cors)
```

Comment:
I think `cors` should be just after `canonical_host` and `force_ssl`. I don’t think we want to respond to requests without going through these two plugs first.